src/components/components.module.ts

```diff
@@ -18,6 +18,9 @@ import { TickIndicatorComponent } from './tick-indicator/tick-indicator';
 import { ModalAlertTitleComponent } from './modal-alert-title/modal-alert-title';
 import { ModalReturnButtonComponent } from './modal-return-button/modal-return-button';
 import { ErrorMessageComponent } from './error-message/error-message';
+import { D255Component } from '../pages/office/components/d255/d255';
```

Comment:
These all need removing when you do the next bit (remove the fields from the office page) as they won't technically be shared.

Reply:
They will be shared between the `pass-finalisation` page and the new page introduced before the green screen.

Reply:
Ok , can we move the components into the root components folder then